pkg/action/bootentries.go

```diff
@@ -0,0 +1,212 @@
+package action
+
+import (
+	"bufio"
+	"fmt"
+	"github.com/erikgeiser/promptkit/confirmation"
+	"github.com/erikgeiser/promptkit/selection"
+	"github.com/kairos-io/kairos-agent/v2/pkg/config"
+	"github.com/kairos-io/kairos-agent/v2/pkg/constants"
+	v1 "github.com/kairos-io/kairos-agent/v2/pkg/types/v1"
+	"github.com/kairos-io/kairos-agent/v2/pkg/utils"
+	fsutils "github.com/kairos-io/kairos-agent/v2/pkg/utils/fs"
+	"github.com/kairos-io/kairos-agent/v2/pkg/utils/partitions"
+	"os"
+	"path/filepath"
+	"strings"
+	"syscall"
+)
+
+func SelectBootEntry(cfg *config.Config, entry string) error {
+	// Read the systemd-boot conf file
+	if !strings.HasSuffix(entry, ".conf") {
+		entry = entry + ".conf"
```

Comment:
This is in case the user selects an entry without the conf by using the --select flag, for example by calling it with active or passive.

Nicer UX imho.

Reply:
gotcha, thanks for the explanation ... how about we make a function comesFromFlag() or something like that, or a comment is also good

Reply:
I dont think we can know if it comes from a flag or not, not I think its the function work to know if that comes from whatever source, but this is mainly sanitation just-in-case.

I'll add a comment there.